### ISSUE TITLE
Can you use a wildcard for keyvaultobjectnames?

### ISSUE THREAD

**Issue by erick-thompson** [Azure/kubernetes-keyvault-flexvol#73]
The pattern that our services follow is that every service has its own Azure Key Vault instance, so the service should have access to all the secrets in the Key Vault. This allows for new secrets to be added to the Key Vault and get picked up by the service only a redeploy (and not needing to modify any config files).

Is there some way to use a wildcard for keyvaultobjectnames?

Thanks,
Erick

**Comment by ritazh** [comment]
@erick-thompson With a wildcard for `keyvaultobjectnames`, do you mean we will then mount all objects in key vault satisfying this pattern? We do not have this feature today but let's keep this issue opened so people can vote for it.

**Comment by erick-thompson** [comment]
@ritazh That is correct. I am mainly looking for secrets, but could see it's usefulness for other Key Vault types as well.
